seed/challenges/02-javascript-algorithms-and-data-structures/object-oriented-programming.json

```diff
@@ -318,7 +318,8 @@
         ""
       ],
       "tests": [
-        "assert(ownProps.includes('name') && ownProps.includes('numLegs'), 'message: <code>ownProps</code> should include the values <code>\"numLegs\"</code> and <code>\"name\"</code>.');"
+        "assert(ownProps.includes('name') && ownProps.includes('numLegs'), 'message: <code>ownProps</code> should include the values <code>\"numLegs\"</code> and <code>\"name\"</code>.');",
```

Comment:
itional review comment
+        "assert(panda) && ownProps.includes('2Legs'), 'message: <code>ownProps</code> black and white <code>\"2Legs\"</code> and <code>\"panda\"</code>.');",
 
Leave a comment